doc/contributing/repairnator-docker-image.md

```diff
@@ -36,3 +37,5 @@ be used.
 If not changed in configuration file, repairnator will use the default
 docker image, `spirals/repairnator:latest`. So any changes that have
 been made locally will not be used.
+
+If you do not upload  your image to docker hub you should also remove the ` docker pull $DOCKER_TAG` from the scripts. 
```

Comment:
Did you test the scripts without it and they worked?

Reply:
Yes it will fetch the local image you just  built.
But I think we should consider a better solution than commenting out part of the script, we could an option in repairnator.cfg.

Reply:
The alternative you said seems nice. Could you open an issue in the repo about this? Thus you or someone else can propose changes in the code about this :)